api/v1/mongodbcommunity_types.go

```diff
@@ -471,6 +471,10 @@ type MongoDBUser struct {
 	// +optional
 	ConnectionStringSecretName string `json:"connectionStringSecretName,omitempty"`
 
+	// ConnectionStringSecretNamespace is the namespace of the secret object created by the operator which exposes the connection strings for the user.
+	// +optional
+	ConnectionStringSecretNamespace string `json:"connectionStringSecretNamespace,omitempty"`
```

Comment:
I think this has some implications with the permissions the operator needs to have when setting this namespace to something other than the default. Do we need to to add special RBAC settings just for this, or re-use the guidance we have for adding rbac settings for different namespaces:
```
kubectl apply -k config/rbac --namespace <my-namespace>
```
This should be documented.

Reply:
Yes, that sounds good to me.

Reply:
Added https://github.com/mongodb/mongodb-kubernetes-operator/pull/1463/commits/d680ab0b4451245690fb27cbd22bb7413ef8e1f3